x-pack/plugins/ml/public/application/timeseriesexplorer/components/plot_function_controls/plot_function_controls.tsx

```diff
@@ -30,11 +34,57 @@ const plotByFunctionOptions = [
 export const PlotByFunctionControls = ({
   functionDescription,
   setFunctionDescription,
+  selectedDetectorIndex,
+  selectedJobId,
+  selectedEntities,
 }: {
   functionDescription: undefined | string;
   setFunctionDescription: (func: string) => void;
+  selectedDetectorIndex: any;
```

Comment:
should be a `number` I guess
```suggestion
  selectedDetectorIndex: number;
```

Reply:
Updated here https://github.com/elastic/kibana/pull/83507/commits/607d7880884564514d8fb34b4972bc4ab644ea75